src/wallet.cpp

```diff
@@ -4209,8 +4209,8 @@ void CWallet::AutoCombineDust()
             vRewardCoins.push_back(out);
             nTotalRewardsValue += out.Value();
 
-            // Combine to the threshold and not way above
-            if (nTotalRewardsValue > nAutoCombineThreshold * COIN)
+            // Combine until our total is enough above the threshold to remain above after adjustments
+            if ((nTotalRewardsValue-nTotalRewardsValue/10) > nAutoCombineThreshold * COIN)
```

Comment:
nit: I think we usually put spaces around '-' and '/' operators so ``if ((nTotalRewardsValue - nTotalRewardsValue / 10) > nAutoCombineThreshold * COIN)`` would be preferred here.

Reply:
Understood and apologies.  I'm usually good about determining the style; but I made this change through the github browser... as I'm rolling it up from my original fix in a distant child of PivX...  So I'm new here and didn't get a chance to establish a pivx dev area for myself.

Should I modify and resubmit, or something for when someone is next in the module?

Reply:
No problem, I'm being picky about it. If you can amend your commit and force push it, it would be the best way to go.

Reply:
No worries; I'm a stickler in my IRL job when people don't stick to the established coding standard; so completely understood.  Looks like I was able to recommit the file (I think)